pkgs/by-name/kr/kraken2/package.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitHub, rsync, perl, wget, gnused, findutils, libgcc, llvmPackages, zlib }:
+
+stdenv.mkDerivation rec {
```

Speaking of `finalAttrs`:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```